src/components/AppFooter.tsx

```diff
@@ -20,7 +20,7 @@ export default function AppFooter() {
           </Grid>
         </Grid>
       </Grid>
-      <Grid container justifyContent="flex-end" xs={2}>
+      <Grid container justifyContent="flex-end" item xs={2}>
```

Comment:
opsie?

grid som container och item samtidigt?

Reply:
I will refactor this component later, when I figured out a good design for the footer